cloud/passportauth/src/auth/PassportAuth.ts

```diff
@@ -62,7 +63,8 @@ export class PassportAuth implements EndpointSecurity {
    * @param sessionOpts - Session options to be used by express-session
    */
   public init(app: express.Express, sessionOpts: SessionOptions) {
-    this.log.info('Initializing express app to use express session and passport');
+    logger.info('Initializing express app to use express session and passport',
+      {level: 'INFO', tag: 'cloud:passportauth:src:auth', src: 'PassportAuth.ts'});
```

Comment:
Same as above avoid adding level src. Tag is ok. This is added by bunyan anyway.